src/Pim/Bundle/UIBundle/Resources/public/less/lib/bootstrap.switch.less

```diff
@@ -2,28 +2,121 @@
 
 /* Overrides src/Pim/Bundle/UIBundle/Resources/public/lib/bootstrap-switch/bootstrap.switch.css */
 .has-switch {
-  @switchHeight: 32px;
+  @switchSize: @AknDefaultButtonSize;
+  @borderWidth: 1px;
 
-  height: @switchHeight;
-  border-color: @AknBorderColor;
-  border-radius: 2px;
+  border-radius: 100px;
+  height: @switchSize;
+  width: @switchSize * 2;
+  border: none;
+  overflow: initial;
 
-  .switch-on,
-  .switch-off,
-  label.switch-small,
-  span.switch-small {
-    height: @switchHeight - 2px;
-    line-height: @switchHeight - 2px;
-    padding-bottom: 0;
-    padding-top: 0;
+  label,
+  label:active {
+    background: none;
+    width: 3 * @switchSize;
+    border: none;
+    margin-left: -@switchSize;
+
+    &:hover {
+      background: none;
+    }
+
+    &:after {
+      content: ' ';
+      background: white;
+      height: @switchSize - 2 * @borderWidth;
+      width: @switchSize - 2 * @borderWidth;
+      border-radius: 100px;
+      position: absolute;
+      top: @borderWidth + 1px;
+      left: @switchSize + @borderWidth;
+    }
   }
 
   &.switch-small {
-    min-width: 100px;
+    min-width: @switchSize * 2;
   }
 
-  label,
-  span {
-    width: 100% / 3;
+  .switch-off,
+  .switch-on {
+    height: @switchSize;
+    margin-left: @switchSize;
+    width: 100%;
+  }
+
+  & > div {
+    transition: all 0.2s ease !important;
+    border-radius: 100px;
+    display: flex;
+  }
+
+  .switch-off {
+    background: lighten(@AknSlateGrey, 20%);
+  }
+
+  .switch-on {
+    background: @AknGreen;
+    margin-left: 0;
+    padding-left: 50%;
+  }
+
+  &.deactivate {
+    cursor: not-allowed !important;
+
+    label:hover {
+      color: white;
+      cursor: not-allowed !important;
+    }
+
+    span {
+      cursor: not-allowed !important;
+    }
+  }
+
+  span.switch-primary,
```

Comment:
Yes because the original bootstrap switch use this (or I should use !important everywhere).
It's just a copy paste of a block from the origina library